Code/Database/AppveyorScriptDatabase.bat

```diff
@@ -13,3 +13,5 @@ SET PASSWORD="Password12!"
 SET INPUT=%cd%\SQLCreateQuery.sql
 
 %SQLCMD% -S%SERVER% -d%DB% -U%LOGIN% -P%PASSWORD% -i%INPUT% -b
+SET INPUT=%cd%\InsertScript.sql
```

Comment:
UI shows "Failed borrow Someone else was faster than you" but it failed becouse the limit was reached